multi-cloud/README.md

```diff
@@ -516,18 +523,93 @@ cd $EXAMPLE_SELDON/workflows
 ```
 
 We already have a port-forward of the ambassador service, which in addition to exposing the Web UIs also exposes the model serving.
-If you want to access the ambassador from inside Kubeflow (e.g. Jupyter), just use the ambassador hostname.
 
+If you want to access the ambassador from inside Kubeflow (e.g. Jupyter), just use the Ambassador hostname (e.g. "ambassador") which is essentially the same as `kubectl get svc/ambassador -n kubeflow -o json | jq -r .spec.clusterIP`.
 
 The deployment name is based on your pipeline, ours is `mnist-classifier`.
-Now you can curl by hitting the REST endpoint:
+
+You can query the Seldon deployments with the following:
+
+```bash
```

Comment:
YDY but
```bash
$ kubectl get seldondeployment -n kubeflow
mnist-classifier
```
becomes:

```bash
kubectl get seldondeployment -n kubeflow
```

Since we trained a mnist classifier we expect:

```
mnist-classifier
```